src/Template/Sentences/of_user.ctp

```diff
@@ -105,26 +105,21 @@ $this->set('title_for_layout', $this->Pages->formatTitle($title));
         <md-toolbar class="md-hue-2">
             <div class="md-toolbar-tools">
                 <h2><?= $this->Paginator->counter($title . ' ' . __("(total {{count}})")); ?></h2>
+
+                <?php 
+                    $options = array(
+                        array( 'param' => 'modified', 'direction' => 'desc', 'label' => __('Most recently updated')),
+                        array( 'param' => 'modified', 'direction' => 'asc', 'label' => __('Least recently updated')),
+                        array( 'param' => 'created', 'direction' => 'desc', 'label' => __('Newest first')),
+                        array( 'param' => 'created', 'direction' => 'asc', 'label' => __('Oldest first'))
+                    );
+                    echo $this->element('sort_menu', array('options' => $options));
+                ?>
+
             </div>
         </md-toolbar>
 
         <md-content layout-padding>
-        <div class="sortBy">
-            <strong><?php echo __("Sort by:") ?> </strong>
-            <?php
-            if ($onlyOriginal) {
-                $urlOptions = $this->Paginator->generateUrlParams(
-                    array('?' => array('only_original' => ''))
-                );
-                $this->Paginator->options(array('url' => $urlOptions));
-            }
```

Comment:
On the page for the sentences of an user, there is an option in the sidebar which allows you to only show original sentences. The old behavior was that this option kept its status when you changed the sort mode. So I think you shouldn't remove this part but move it before the `$options` array (line 110).

Reply:
I restored those lines. I hope those options are also applid to generateUrl function.

Reply:
Yes, `Paginator->options()` sets the default options for every Paginator method called afterwards.